### PR TITLE
TSK-229: Add two custom actions.

### PR DESCRIPTION
**Pull request by YakirOren**

### CODE REVIEW

**Review by ozhey — APPROVED** [review]

File: custom_actions/actions/AddGroupMembers.action.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: "AddUserToGroup"
+description: "Adds members to Group"
+enabled: true
+parameters:
+  "GroupKey":
```
Comment (shirleygordon-blink) [inline custom_actions/actions/AddGroupMembers.action.yaml:5]: Group Key

File: custom_actions/actions/DeleteMultipleGroupMembers.action.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "DeleteUserFromGroup"
+description: "Delete Multiple Group Members"
+enabled: true
+parameters:
+  "GroupKey":
```
Comment (shirleygordon-blink) [inline custom_actions/actions/DeleteMultipleGroupMembers.action.yaml:5]: Group Key

File: custom_actions/actions/DeleteMultipleGroupMembers.action.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    type: string
+    description: "Identifies the group in the API request. The value can be the group's email address, group alias, or the unique group ID"
+    required: true
+    index: 1
+  "MemberKeys":
```
Comment (shirleygordon-blink) [inline custom_actions/actions/DeleteMultipleGroupMembers.action.yaml:10]: Member Keys

File: mask.yaml [hunk 4]
```diff
@@ -97,29 +97,27 @@ actions:
...
+    alias: ListGroupMembers
     parameters:
       groupKey:
-        alias: GroupKey
-        index: 1
-        required: true
-      email:
-        alias: Email
-        index: 2
+        alias: "GroupKey"
```
Comment (shirleygordon-blink) [inline mask.yaml:104]: Group Key

File: mask.yaml [hunk 5]
```diff
@@ -97,29 +97,27 @@ actions:
...
-        required: true
-      memberKey:
-        alias: MemberKey
+      roles:
+        alias: "Roles"
+        description: "The `roles` query parameter allows you to retrieve group members by role. Allowed values are `OWNER`, `MANAGER`, and `MEMBER`."
         index: 2
-        required: true
+      includeDerivedMembership:
+        alias: "IncludeDerivedMembership"
```
Comment (shirleygordon-blink) [inline mask.yaml:113]: Include Derived Membership

File: mask.yaml [hunk 6]
```diff
@@ -97,29 +97,27 @@ actions:
...
+        description: "Whether to list indirect memberships. Default: false."
+        index: 3
+      maxResults:
+        alias: "MaxResults"
```
Comment (shirleygordon-blink) [inline mask.yaml:117]: Max Results

File: mask.yaml [hunk 7]
```diff
@@ -97,29 +97,27 @@ actions:
...
+        description: "Maximum number of results to return. Max allowed value is 200."
+        index: 4
+      pageToken:
+        alias: "PageToken"
```
Comment (shirleygordon-blink) [inline mask.yaml:121]: Page Token

File: mask.yaml [hunk 8]
```diff
@@ -97,29 +97,27 @@ actions:
...
+        description: "Token to specify next page in the list."
+        index: 5
```
Comment (shirleygordon-blink) [inline mask.yaml:123]: new line

File: mask.yaml [hunk 9]
```diff
@@ -97,29 +97,27 @@ actions:
         alias: UserKey
         required: true
 
-  directory.members.insert:
-    alias: AddUserToGroup
+  directory.members.list:
+    alias: List Group Members
```
Comment (shirleygordon-blink) [inline mask.yaml:101]: action alias shouldn't have spaces

**Review by shirleygordon-blink — APPROVED** [review]

**Pull request merged**